scripts/translationScripts/update-en-ts.py

```diff
@@ -23,9 +24,16 @@ def fixupTranslations(enTsFile: str):
 
 
 if __name__ == "__main__":
-    p = subprocess.run(['lupdate', '../../ui/nim-status-client.pro', '-target-language', 'en'],
+    # full base TS file (has to come first as we're targetting the same language)
+    basefile = "../../ui/i18n/qml_base.ts"
+    if os.path.exists(basefile):
+        os.remove(basefile) # we don't "merge" here, always overwrite and copy/paste translation
```

Comment:
Why does it need to be deleted first? Isn't `lupdate` going to overwrite `qml_base.ts` anyway?